_digest/october-2020/an-effective-guide-to-bullet-journalling.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: "An Effective Guide to Bullet Journaling for Students"
+date: 10/04/2020
+releaseBatch: October 2020
+summary: In recent years, an abrupt boom in studygrams on social media has taken over the productivity world. Although aesthetic note taking, stationery, and handwriting go through on-and-off phases of popularity, one key aspect to an aesthetically productive life has always been trending- bullet journaling. 
+authors: ["Deeksha Yelamanchi"]
+tags: ["Creativity", "Journaling", "Student Life"]
+---
+<div>
+    <p><strong>How to organize your time creatively- An effective guide to bullet journaling for students</strong></p>
+    <p>In recent years, an abrupt boom in studygrams on social media has taken over the productivity world. Although aesthetic note taking, stationery, and handwriting go through on-and-off phases of popularity, one key aspect to an aesthetically productive life has always been trending- bullet journaling.&nbsp;</p>
+    <p>Those who are new to bullet journaling struggle to devise how to properly use the technique to organize their schedule and to-do lists in order to reap optimal results of a simultaneous creative outlet and effective organization system. But, knowing the right basics, bullet journaling can help one optimize their time to lead both a productive and healthy life. Today, I will be sharing my guide to bullet journaling 101, and my tips for achieving the above ideal!</p>
+    <p>Before starting to bullet journal, it is crucial for one to find a bullet journal that suits their style and their needs. Not only will it help with creating beautiful bullet journal spreads, but it can also aid with an effective planning system, as you should be comfortable with using the book for long periods of time. The following is a short list of bullet journaling notebooks that I personally find to be most effective (in order of lowest to highest price range).</p>
+    <ol><li><a href="https://www.amazon.com/Dotted-Bullet-Grid-Journal-Notebook/dp/B07FD7FSW3"><strong>Plouma Dotted Grid Notebook</strong></a><strong> ($7.79 CAD)</strong></li></ol>
```

Comment:
Take a look at how the other blog posts add links and follow their example for all the links in this blog post
```
<a href="https://learningcommons.ubc.ca/mental-health-and-balancing-student-work-life/" target="_blank" rel="noopener noreferrer">Mental Health and Balancing Student Work Life</a>
``` 
from https://raw.githubusercontent.com/PuMProfessionals/pump-web-v2/main/_digest/october-2020/incorporate-social-life-in-student-life.mdx 
 (add the `target=` and `rel=` attributes)